index.js

```diff
@@ -117,10 +117,18 @@ var login = function login(apiOptions, username, pass, orderedHeaders, clientUse
         return reject(err || res);
       }
 
+      var details = parseDetails(res);
+      if (details === 'TWO_FA_REQUIRED'){
+        resolve(details);
+      }
+
       var errors = parseErrors(res);
       if (errors.length) {
         if (Array.isArray(errors[0]) && !!errors[0][0]) {
           var firstError = errors[0][0];
+          if (firstError === 'WRONG_PASSWORD' && errors[0].length > 2 && errors[0][2] === 'otp') {
+            return reject('WRONG_OTP');
```

Comment:
We could also add a WRONG_OTP error to the desktop login form instead of adding a weird special case here.